### ISSUE TITLE
`show-open-prs-of-forks`: select_dom(...) is undefined

### ISSUE THREAD

**Issue by reegnz** [refined-github/refined-github#5373]
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.
- [X] The bug happens after clearing extension cache. The "Clear cache" button can be found at the bottom of the options page.

### Example URLs

https://github.com/reegnz/tfenv

### Extension version

22.2.1

### Browser(s) used

Firefox

### Description

```
init@moz-extension://4e8cfbf6-90a4-42db-82e5-b5ed4b740f55/build/refined-github.js:8354:19
```

When I disable the cache in the Network tab of the inspect age of Firefox, it shows up every once in a while, but not always. I didn't manage to reproduce further.

### Include in this issue:

- [ ] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

**Comment by kidonng** [comment]
> When I disable the cache in the Network tab of the inspect age of Firefox

Is this really the cause? Because `select-dom` is a dependency bundled into the extension and should have nothing to do with your network.

Have you tried to reinstall the extension, in case it may be corrupted for some reason?

**Comment by fregante** [comment]
@kidonng the cache might affect the loading speed of the page and a selector might not be ready in time. Maybe it’s a race condition.

**Comment by kidonng** [comment]
Oh well, I thought author is saying `select_dom` is undefined. 😅

**Comment by fregante** [comment]
I'm gonna assume this has the same cause as #5435 since the selector similarly expects a relative attribute.

It was fixed in [`a067bd2` (#5640)](https://github.com/refined-github/refined-github/pull/5640/commits/a067bd218e867a1f6f486c52d1e6bc5a1e2a7358)
